### PR TITLE
Paf 253: Resolve PAF deployment Issues

### PR DESCRIPTION
**Pull request by adityababumallisettiHO**
## What?
Resolve PAF Prod Issues 
PAF prod URL 404 error
PAF acme pods created due to invalid certs issued by clusterissuer

## Why?
ACP documentation suggests us to use nginx external as Ingress class for external access of ingress
https://docs.acp.homeoffice.gov.uk/how-to/ingress/

## How?
removed conditional statements restricting nginx-external ingress class to non-prod env. Now All the external ingress are accessible outside VPN

### CODE REVIEW

File: kube/file-vault/file-vault-ingress.yml [hunk 1]
```diff
@@ -9,9 +9,7 @@ metadata:
   {{ end }}
 {{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS | indent 2 }}
 spec:
-  {{ if or (eq .KUBE_NAMESPACE .BRANCH_ENV) (eq .KUBE_NAMESPACE .UAT_ENV) }}
   ingressClassName: nginx-external
```
Comment (jamiecarterHO) [inline kube/file-vault/file-vault-ingress.yml:12]: @adityababumallisettiHO Just a note - also need to make this change in `kube/app/ingress-external.yml`

Comment (adityababumallisettiHO) [reply kube/file-vault/file-vault-ingress.yml:12]: Good one. I didn't notice this one. Thanks Jamie. I have updated now.

**Review by jamiecarterHO — APPROVED** [review]
LGTM - remember that when this is promoted, assuming the certs work ok, that this will be open to the internet if there is no whitelist for PROD env in annotations.

**Comment by adityababumallisettiHO** [comment]
I am closing this pull request without merging and deleting branch as this prod url is resolved.

**Pull request closed without merging**